### DISCUSSION TITLE
How do I change from 2,4 GHz to 5 GHz Wlan on RPI 4

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#657] [General]
My rpi 4only connects with 2.4 GHz .

**Comment by phreatic62** [comment]
Have you tried a (slightly crappy) workaround that Alexa users use? Turn off 2.4G on your router while you make the connection.

**Comment by R101G** [comment]
Try setting the router to have a separate SSID for 5GHz and change the ESSID entry in the /etc/netcl file to match it.

**Comment by hjfisch** [comment]
Tried both but no luck.
No probs with volumio .

**Comment by R101G** [comment]
It works OK for me. The country code restricts channels, though, so that might be worth checking. Default is 00.

My Pi 4 does not show DFS channels with the onboard wifi, which I assume is due to the firmware. I don't know if different kernels load different firmware, but it might be worth checking your router is set to ch36, or something equally universal.

**Reply by hjfisch** [reply]
Setting to Channel 36 and seperate SSID's are working now in 5 GHz .
Thank's

**Comment by R101G** [comment]
Glad it's working, but are you sure it worked in Volumio? It looks as if Pi 4 onboard wifi is limited to Chinese channels, so if you have an Asus router which defaults to channel 100 there is no chance of it connecting.

**Reply by hjfisch** [reply]
I've been using Volumio for 2 years now with same SSID for 2.4 and 5 GHz and Channel 100 now .
No probs with upper Channels . Volumio is switching automaticly to 5 GHz connection .
Same with moOde Player . I think it belongs to the wlan-driver , which needs some work on in 
rAudio-1 . 
Due to the fact of playing from USB-CD and showing the albumcovers in webradio , I think that I will stay with rAudio . Also it seems to be quite stable now with the latest version .

**Comment by R101G** [comment]
That's interesting. I think Volumio is Debian based, and Moode is Rasbian, so it might be that Arch Linux is loading restricted firmware into the wifi chip.

**Reply by hjfisch** [reply]
Volumio 2 with 240 Gb/s , Volumio 3 with 433 Gb/s , Moode with 433 Gb/s  and 5 GHz  Channel 100 .
Moode has a crappy Interface and Volumio is rather expensive if you want to use CD-Playback .
